flow-typed/types/source.js

```diff
@@ -0,0 +1,5 @@
+declare type Source = {
```

Comment:
No point in defining a source as an object or null.
If you intend to nullify it in your flow then use `?Source` where you hold your sources